scripts/gui/microbe_editor.mjs

```diff
@@ -1169,6 +1186,9 @@ function processPatchMapData(data){
 function updateSelectedPatchData(patch){
     selectedPatch = patch;
 
+    // Reset species shown
+    document.getElementById("speciesInPatch").textContent = "";
```

Comment:
could be that textContent do not work down where species are written because here should be `innerHTML = ""`

Reply:
I recommend you use chrome to view what happens and especially check the DOM tree to see if all the elements get created and also the inspector to see what takes up space where.

Reply:
I check what happens because this is the last thing is not gonna work, when we use textContent the result of the div is this:

` <div id="speciesInPatch">Primum Thrivium with populatiuon 130Virodsia Noyes with populatiuon 1431<br></div>`

If we use innerHTML just in the reset of div, or where we add each species it's the same.
If we use innerHTML for each species what there is in the reset part don't change anything.

I think the reason is that textContent do not allow to put a tag br in the middle of the text, he he put the br always after the text that is in the div.

Reply:
I think that if the result was `<div id="speciesInPatch">Primum Thrivium with populatiuon 130Virodsia Noyes with populatiuon 1431</div><br>` (also note the typos in population) it would work. I think if the br is in the div CSS margin collapsing might apply so it disappears.
So a br element should be appended to the container where the div is put in.